src/MasonryFlashList.tsx

```diff
@@ -117,20 +129,24 @@ function MasonryFlashListImpl<R>(
     [_contentContainerStyle, contentContainerStyle.paddingTop]
   )
 
+  const refWorkaround = useCallback(
+    (value: MasonryFlashListMemoRef | null): void => {
+      // https://github.com/Shopify/flash-list/blob/2d31530ed447a314ec5429754c7ce88dad8fd087/src/FlashList.tsx#L829
+      // We are not accessing the right element or view of the Flashlist (recyclerlistview). So we need to give
+      // this ref the access to it
+      ;(recyclerRef as any)(value?.recyclerlistview_unsafe)
```

Comment:
this was not working properly before the patch and exporting it from the interface fixes the rendering issue! 🔥

Reply:
this and the recyclerRef + useEffect recyclerRef additions